Gruntfile.js

```diff
@@ -34,42 +34,8 @@ module.exports = function(grunt) {
     },
     nodeunit: {
       tests: [
-        "test/bootstrap.js",
-        "test/board.js",
-        "test/board-connection.js",
-        "test/compass.js",
-        "test/options.js",
-        "test/board.pins.js",
-        "test/board.component.js",
-        "test/capabilities.js",
-        // ------------------
-        "test/accelerometer.js",
-        "test/animation.js",
-        "test/button.js",
-        "test/esc.js",
-        "test/fn.js",
-        "test/gyro.js",
-        "test/imu.js",
-        "test/lcd.js",
-        "test/led.js",
-        "test/ledcontrol.js",
-        "test/motor.js",
-        "test/pin.js",
-        "test/piezo.js",
-        "test/ping.js",
-        "test/pir.js",
-        "test/proximity.js",
-        "test/reflectancearray.js",
-        "test/relay.js",
-        "test/repl.js",
-        "test/sensor.js",
-        "test/servo.js",
-        "test/shiftregister.js",
-        "test/sonar.js",
-        "test/stepper.js",
-        "test/temperature.js",
-        "test/switch.js",
-        "test/wii.js"
+        "test/bootstrap/*.js",
+        "test/*.js"
```

Comment:
@rwaldron Let me make sure I understand—it sounds like my assumption 2 is incorrect (that all tests in `test` should be runnable). Would it be acceptable to create a place you could chuck your tests (i.e. another subdir)? I see in the referenced thread that a naming convention isn't gonna fly? Is the only way forward here to go back to listing each test file explicitly?